### DISCUSSION TITLE
Mewn vs package "embed"

### DISCUSSION THREAD

**Discussion by GeekStocks** [wailsapp/wails#590] [Ideas]
Does the release of golang v1.16 this February with its new [package "embed"](https://tip.golang.org/pkg/embed/) spell the end of the Mewn dependency in Wails for v2? Is it something you are thinking about?

**Comment by leaanthony** [comment]
Mewn is already not a dependency for v2 😄 Though embed doesn't help us either as it's not dynamic.
